vignettes/irecist.Rmd

```diff
@@ -0,0 +1,858 @@
+---
+title: "Creating ADRS with iRECIST endpoints"
+output: 
+  rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Creating ADRS with iRECIST endpoints}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+
+library(admiral)
+
+link <- function(text, url) {
+  return(
+    paste0(
+      "[", text, "]",
+      "(", url, ")"
+    )
+  )
+}
+dyn_link <- function(text,
+                     base_url,
+                     relative_url = "",
+                     # Change to TRUE when admiral adopts multiversion docs
+                     is_multiversion = FALSE,
+                     multiversion_default_ref = "main") {
+  url <- paste(base_url, relative_url, sep = "/")
+  if (is_multiversion) {
+    url <- paste(
+      base_url,
+      Sys.getenv("BRANCH_NAME", multiversion_default_ref),
+      relative_url,
+      sep = "/"
+    )
+  }
+  return(link(text, url))
+}
+# Other variables
+admiral_homepage <- "https://pharmaverse.github.io/admiral/cran-release"
+
+library(admiraldev)
+```
+
+# Introduction
+
+This article describes creating an `ADRS` ADaM with oncology endpoint parameters
+based on iRECIST. It shows a similar way of deriving the endpoints presented in 
+[Creating ADRS (Including Non-standard Endpoints)](adrs.html). Most of the endpoints 
+are derived by calling `admiral::derive_extreme_event()`. Thus the examples in this 
```

Comment:
I would remove the sentence "Thus...". I would not use iRecist as a starting point for Recist 1.1

Reply:
will update